lightning/src/routing/router.rs

```diff
@@ -149,9 +161,31 @@ impl cmp::PartialOrd for RouteGraphNode {
 struct DummyDirectionalChannelInfo {
 	cltv_expiry_delta: u32,
 	htlc_minimum_msat: u64,
+	htlc_maximum_msat: Option<u64>,
 	fees: RoutingFees,
 }
 
+#[derive(Clone)]
+struct RouteHopInGraph {
+	route_hop: RouteHop,
+	available_msat: Option<u64>,
+	/// Minimal fees required to route to the source node of this hop (one hop "before" using this hop).
+	/// Useful to compute the cost of reaching the source of this hop, which can be later used to choose
+	/// the cheapest, while we traverse backwards.
+	src_lowest_inbound_fees: RoutingFees,
+	/// Fees of the channel used in this hop.
+	channel_fees: RoutingFees,
+	/// Consists of the fee of this channel (see channel_fees), the fee paid to reach this hop (see src_lowest_inbound_fees)
+	/// and all the fees paid *after* this channel on the way to the destination (because we traverse backwards).
```

Comment:
Saying "we traverse backwards" doesn't really help, forward/backward compare to what ?